packaging/debian/postinst

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -e
+# post-install script for cvmfs-config-osg
+#
+# see: dh_installdeb(1)
+
+# summary of how this script can be called:
+#        * <postrm> `remove'
+#        * <postrm> `purge'
+#        * <old-postrm> `upgrade' <new-version>
+#        * <new-postrm> `failed-upgrade' <old-version>
+#        * <new-postrm> `abort-install'
+#        * <new-postrm> `abort-install' <old-version>
+#        * <new-postrm> `abort-upgrade' <old-version>
+#        * <disappearer's-postrm> `disappear' <overwriter>
+#          <overwriter-version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+if deb-systemd-helper --quiet is-enabled autofs.service; then
+    deb-systemd-invoke try-reload-or-restart autofs.service
+fi
```

Comment:
Likewise, on Redhat the autofs reload is part of the installation instructions after manual creation of `/etc/auto.master.d/cvmfs.autofs`, not part of any cvmfs package.  The same for postrm.

If you want to make it easy for the system administrator, there could be a separate package called `cvmfs-autofs` or `cvmfs-autofs-config` which we could put in the cvmfs-contrib repository.  I suggest making a CVM ticket for that creating it in a new github repo.  Then if Jakob agrees he can import that into the cvmfs github project and I can build it for the cvmfs-contrib repository.